### PR TITLE
Adds version in describe command of task

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
- This patch adds a version field in the
    describe command of task

 - For e.g.

```
╰─ tkn task describe
Name:          buildah
Namespace:     default
Description:   Buildah task builds source into a container image and then pushes it to a container registry.
Buildah Task builds source into a container image using Project Atomic's Buildah build tool.
Version:   0.2     <------- Version of the resource
```

- If the resource doesn't have a version then it won't show 
 the version field in the output of describe command

Fixes : https://github.com/tektoncd/cli/issues/1264
Signed-off-by: Puneet Punamiya <ppunamiy@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

### CODE REVIEW

File: pkg/cmd/task/describe.go [hunk 1]
```diff
@@ -41,6 +41,10 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .Task.Name }}
 {{decorate "bold" "Description"}}:	{{ .Task.Spec.Description }}
 {{- end }}
 
+{{- if ne (formatVersion .Task.Labels) ""}}
```
Comment (piyush-garg) [inline pkg/cmd/task/describe.go:44]: The name should be `findVersion` better I think

Comment (piyush-garg) [inline pkg/cmd/task/describe.go:44]: Should we just get the value once, and use it both places, inspite of calling the func twice

File: pkg/formatted/version_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import (
+	"testing"
+
+	"github.com/tektoncd/cli/pkg/test"
+)
+
+func TestVersion(t *testing.T) {
+	version := make(map[string]string)
+
+	version["app.kubernetes.io/version"] = "v1"
```
Comment (vinamra28) [inline pkg/formatted/version_test.go:26]: can we add one more test with other label to check whether it returns an empty string or not? eg `tekton.dev/name: foobar`

**Review by vinamra28 — COMMENTED** [review]
just one review otherwise looks good to me

**Review by vinamra28 — APPROVED** [review]
/lgtm
/woof

**Review by vdemeester — APPROVED** [review]
/cc @tektoncd/cli-maintainers

**Review by vinamra28 — COMMENTED** [review]
/lgtm
